habitat-lab/habitat/sims/habitat_simulator/sim_utilities.py

```diff
@@ -137,6 +138,8 @@ def bb_ray_prescreen(
     obj: habitat_sim.physics.ManagedRigidObject,
     support_obj_ids: Optional[List[int]] = None,
     check_all_corners: bool = False,
+    estimate_support_stability: bool = True,
```

Comment:
NOTE:
This seems to help in edge cases (as shown in the videos) with very high sample numbers.
However, in practice with typical scenarios it doesn't do much and actually leads to slightly worse generator performance.